### PR TITLE
Added anchors and links to UX and data on department page

### PR DESCRIPTION
**Pull request by jsoftwareengineering**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Two times the same link in the Wikidata section?

**Comment by jsoftwareengineering** [comment]
No, but I did get the label wrong. One should say Technical Wishes team.

**Comment by addshore** [comment]
Deployed

File: app/templates/pages/department.html [hunk 1]
```diff
@@ -344,6 +348,7 @@
 
                     <div class="row">
                         <div class="col-lg-12">
+                            <a name="department-wide"></a>
                             <h3>Department wide</h3>
```
Comment (thiemowmde) [inline app/templates/pages/department.html]: I find `<h3 id="department-wide">Department wide</h3>` best practice.

Comment (jsoftwareengineering) [inline app/templates/pages/department.html]: updated on master

**Pull request merged**